### PR TITLE
[10.x] Fix typo for setUser in `GuardHelpers`

### PR DESCRIPTION
**Pull request by milwad-dev**

### CODE REVIEW

**Review by nunomaduro — APPROVED** [review]
Updated the contract.

File: src/Illuminate/Contracts/Auth/Guard.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function hasUser();
      * Set the current user.
      *
      * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
-     * @return void
+     * @return $this
```
[the remark this answers is not in the document]
Comment (milwad-dev) [reply src/Illuminate/Contracts/Auth/Guard.php:54]: Change branch to `master`?

[the remark this answers is not in the document]
Comment (driesvints) [reply src/Illuminate/Contracts/Auth/Guard.php:54]: Or do this change to the contract on master instead.

[the remark this answers is not in the document]
Comment (milwad-dev) [reply src/Illuminate/Contracts/Auth/Guard.php:54]: Ok, Merge this PR and I add a separate PR for Contract

[the remark this answers is not in the document]
Comment (nunomaduro) [reply src/Illuminate/Contracts/Auth/Guard.php:54]: @driesvints Debatable - as for me, this contract is unusable if you don't return `$this` there.

**Pull request merged**